packages/application-shell/src/components/navbar/menu-items.styles.ts

```diff
@@ -33,24 +34,22 @@ const getContainerPositionBasedOnMenuItemPosition = (
     `,
 ];
 
+// TODO: remove completely as part of the recolouring rollout cleanup process
```

Comment:
Thank you. Sure, because [it's not visible anymore in the new theme](https://www.figma.com/file/faaHnpkH23tLdambqnF1OG/New-colors-V1?type=design&node-id=4169-17409&mode=design&t=lynTfudtPfjw8184-4)

For now I'm simply [hiding it in css in the new theme](https://github.com/commercetools/merchant-center-application-kit/blob/67badd518eab16075e1919e2a108847fb65d1cc3/packages/application-shell/src/components/navbar/menu-items.styles.ts#L45)

Reply:
Oh! I see.
Yes, maybe a _TODO_ also in the JSX would help with this explanarion.

Thanks.